src/main/java/frc/robot/utilities/TOFSensorSimple.java

```diff
@@ -7,27 +7,42 @@
 import com.playingwithfusion.TimeOfFlight;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.button.Trigger;
+import frc.robot.Robot;
 
 /** Add your docs here. */
 public class TOFSensorSimple {
   private TimeOfFlight sensor;
   private double threshold;
+  private boolean isBeamBrokeSim;
 
   public TOFSensorSimple(int ID, double threshold) {
     sensor = new TimeOfFlight(ID);
     this.threshold = threshold; // in mm
+    isBeamBrokeSim = false;
+  }
+
+  public TOFSensorSimple(int ID) {
+    sensor = new TimeOfFlight(ID);
+    this.threshold = 1; // in mm
```

Comment:
1mm is a really small distance.  Should we change this default to something more likely to be usable?